Components/LoginScreen.js

```diff
@@ -3,29 +3,54 @@ import { StyleSheet, TextInput, Text, View, Button, Image, AsyncStorage } from '
 import { createStackNavigator } from 'react-navigation';
 import logo from './images/logo_planted.png';
 import { connect } from 'react-redux';
-import {submitUserLoginInformation} from './helperFunctions/Login';
+import {submitUserLoginInformation, setLoginToAsyncStorage} from './helperFunctions/Login';
 import {updateToken, updateIsUserLoggedIn} from '../actions';
 
+
 class LoginScreenDumb extends Component {
     constructor(props) {
         super(props);
-        this.state = { email: '', password: '' };
+        this.state = { 
+            email: '', 
+            password: '',
+            failedToLogin: false,
+        };
       }
+    
+    //testing mode/ delete after
+    componentDidMount() {
+        let {navigation, dispatch} = this.props;
+        let userEmail = 'test1@gmail.com';
+        let userPassword = '1111';
+        submitUserLoginInformation(userEmail, userPassword)
+            .then(async (res) => {
+                console.log(res)
+                if (res.status === 200) {
+                    res = await res.text();
+                    setLoginToAsyncStorage(res)
+                    dispatch(updateToken(res));
+                    dispatch(updateIsUserLoggedIn());
+                    navigation.navigate('Profile');
+                }
+            })
+    }
```

Comment:
This also looks like it will navigate directly to the profile page upon component did mount?

Reply:
Again, you can change it to whatever page you're working on. It's there just to save time while developing